be/src/storage/tablet.cpp

```diff
@@ -170,10 +170,14 @@ Status Tablet::revise_tablet_meta(const std::vector<RowsetMetaSharedPtr>& rowset
         StorageEngine::instance()->add_unused_rowset(it->second);
         _rs_version_map.erase(it);
     }
-    for (auto& it : _inc_rs_version_map) {
-        StorageEngine::instance()->add_unused_rowset(it.second);
+    for (auto& [v, rowset] : _inc_rs_version_map) {
+        StorageEngine::instance()->add_unused_rowset(rowset);
+    }
+    for (auto& [v, rowset] : _stale_rs_version_map) {
+        StorageEngine::instance()->add_unused_rowset(rowset);
```

Comment:
inc_rs_rowset and stable_rs_rowset may have duplicate rowset.
It's can not be gc in the reference check.

Reply:
The StorageEnginecan handle this:
`
void StorageEngine::add_unused_rowset(const RowsetSharedPtr& rowset) {
    if (rowset == nullptr) {
        return;
    }

    VLOG(3) << "add unused rowset, rowset id:" << rowset->rowset_id() << ", version:" << rowset->version().first << "-"
            << rowset->version().second << ", unique id:" << rowset->unique_id();

    auto rowset_id = rowset->rowset_id().to_string();

    std::lock_guard lock(_gc_mutex);
    auto it = _unused_rowsets.find(rowset_id);
    if (it == _unused_rowsets.end()) {
        rowset->set_need_delete_file();
        rowset->close();
        _unused_rowsets[rowset_id] = rowset;
        release_rowset_id(rowset->rowset_id());
    }
}
`